wrds/sql.py

```diff
@@ -191,7 +192,10 @@ def load_library_list(self):
 GROUP BY nv.schemaname
 ORDER BY 1;
         """
-        cursor = self.connection.execute(query)
+        if version.parse(sa.__version__) > version.parse("2"):
+            cursor = self.connection.exec_driver_sql(query)
+        else:
+            cursor = self.connection.execute(query)
```

Comment:
This section of code and the one below look identical. In the interest of being DRY, can we refactor this to a function? Future us will thank current us!